README.md

```diff
@@ -15,6 +15,12 @@ You probably want to **[see this page as it was when the latest version was publ
 
 ## Installation
 
+Here are the one-liners for installing pre-compiled `cargo-binstall` binary from release on Linux and acOS:
+
+```
+curl -L https://raw.githubusercontent.com/cargo-bins/cargo-binstall/main/install-from-binstall-release.sh | bash
```

Comment:
Would be good to include curl flags as per rustup.rs for some security

Reply:
Ya, especially if we try to learn to beginners to not download and execute any script blindly from the web on Linux.

Reply:
That's a really good idea!

I've updated the PR to include flags `--proto '=https' --tlsv1.2 -sSf` for every `curl` invocation, including the `README.md` and the `install-from-binstall-release.sh` script.